docker/appserver/WildFly/example/src/scripts/add_modules.py

```diff
@@ -0,0 +1,22 @@
+import os
+# base path to resources, configurations jars
+path_to_jars='/opt/jboss/wildfly/standalone/lib/ext/'
+# name of the module that war file will depend on
+moduleName='module.nl.nn.adapterframework'
+# list of jar files
+jarList = os.listdir(path_to_jars)
+# resources that the module will be created upon
+resources = ''
+resourceDelimiter=','
+# client executable path
+jboss_cli = '/opt/jboss/wildfly/bin/jboss-cli.sh'
+
+for jarFile in jarList:
+	resources += path_to_jars+jarFile+resourceDelimiter
+
+resources=resources[0:-1]
+
+command="module add --name="+moduleName+" --resources="+resources+" --resource-delimiter="+resourceDelimiter
+
+os.system(jboss_cli+" --command='"+command+"'")
+
```

Comment:
If the modules are also in the standalone.xml this isn't required is it?

Reply:
These are resources and configurations that are provided as jars. It is not possible to just put them in the lib folder to make them available in classpath. It is required to add those as modules to the server. This is a way to add them as modules.